modules/valkey/valkey_test.go

```diff
@@ -0,0 +1,157 @@
+package valkey_test
+
+import (
+	"context"
+	"fmt"
+	tcvalkey "github.com/testcontainers/testcontainers-go/modules/valkey"
+	"github.com/valkey-io/valkey-go"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
Proper imports format ( I ran `make lint` from the module)
```suggestion
import (
	"context"
	"fmt"
	"path/filepath"
	"testing"
	"time"

	"github.com/google/uuid"
	"github.com/stretchr/testify/require"
	"github.com/valkey-io/valkey-go"

	tcvalkey "github.com/testcontainers/testcontainers-go/modules/valkey"
)
```